context/app/static/js/components/entitySearch/filters/Filter/enums.js

```diff
@@ -0,0 +1,4 @@
+export const FILTER_TYPES = {
+  multiList: 'multiList',
+  rangeSlider: 'rangeSlider',
+};
```

Comment:
Unless there's going to be a lot more here, I'd just inline this. Jumping between files can be a nuisance.